contracts/libraries/Quotes.sol

```diff
@@ -57,11 +57,9 @@ library Quotes {
         agreedAmount = quote.value + quote.callFee + quote.productFeeAmount + quote.gasFee;
 
 
-        uint delta = agreedAmount / 10000;
-        // transferred amount should not be lower than (agreed amount - delta),
-        // where delta is intended to tackle rounding problems
-        if (agreedAmount - delta > transferredAmount) {
-            revert AmountTooLow(transferredAmount, agreedAmount - delta);
+        // transferred amount should not be lower than agreed amount
+        if (agreedAmount > transferredAmount) {
```

Comment:
this is ok according to the audit, but doesn't consider the rounding issues, when going from sats to wei,the precision differences between both networks might generate a non refundable quote if they aren't considered. My recommendation is to remove the delta as the report says, but implement this check in a similar way to the one we do in pegout https://github.com/rsksmart/liquidity-bridge-contract/blob/master/contracts/LiquidityBridgeContractV2.sol#L718C8-L722C88 in that way, we allow the bitcoin transaction to either round or truncate the wei value, protecting the user from cases where the client app doesn't round the wei -> sat value properly